scripts/linkeyTest.js

```diff
@@ -0,0 +1,71 @@
+import fetch from 'node-fetch';
+
+/* eslint-disable no-undef */
+const getServiceName = serviceConfig => {
+  if (serviceConfig.default) {
+    return service.default.service;
+  }
+
+  const variants = Object.keys(serviceConfig);
+  return service[variants[0]].service;
+};
+
+const getLinks = (serviceConfig, type) => {
+  if (serviceConfig.default) {
+    return serviceConfig.default[type];
+  }
+
+  const variants = Object.keys(serviceConfig);
+  const serviceConfigVariantOne = serviceConfig[variants[0]][type];
+  const serviceConfigVariantTwo = serviceConfig[variants[1]][type];
+
+  return type === 'navigation'
+    ? serviceConfigVariantOne.concat(serviceConfigVariantTwo)
+    : { ...serviceConfigVariantOne, ...serviceConfigVariantTwo };
+};
+
+const getAbsoluteUrl = link => {
+  const regex = /^http(s)*:\/\//;
+  return regex.test(link) ? link : `https://www.bbc.com${link}`;
+};
+
+const fetchResponse = async link => {
+  const fetchStatus = await fetch(link, { timeout: 20000 });
+  return fetchStatus.status;
+};
```

Comment:
What do you think about moving these functions into a separate file?

Reply:
it would clean up the file, but not sure if that's necessary since these functions most likely won't be used anywhere else and are specific to these tests